dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/toolbar.test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/*eslint max-nested-callbacks: 0*/
+define([
+    'underscore',
+    'uiRegistry',
+    'ko',
+    'Magento_Ui/js/grid/toolbar',
+    'Magento_Ui/js/lib/view/utils/dom-observer',
+    'Magento_Ui/js/lib/view/utils/async'
+], function (_, registry, ko, Toolbar, observer, $) {
```

Comment:
Where is jquery defined ? please use correct order definition. and remove unused objects

Reply:
why you just not use` 'jquery'` ?

Reply:
please use `'jquery'`instead  of  `'Magento_Ui/js/lib/view/utils/async'`